eos_data_distribution/ndn/dbus/base.py

```diff
@@ -244,9 +244,13 @@ def _on_request_interest_complete(self, interface, res, interest):
             logger.debug('request-interest got error: %s', error)
             if str(error).find('ETRYAGAIN') != -1:
                 return self._dbus_express_interest(interest)
+            elif error.code == 24: # Timeout
```

Comment:
Can you use `error.matches()` instead? That would allow disambiguating the error domain as well (the code alone is not enough to uniquely identify the error)